README.md

```diff
@@ -324,6 +333,19 @@ Types are exported from `./lib/types/[node/web]/index.d.ts` and should be automa
   });
   ```
 
+  - `topUpWithTokens({ tokenAmount, feeMultiplier })` - Tops up the connected wallet with Credits by submitting a payment transaction for the token amount to the Turbo wallet and then submitting that transaction id to Turbo Payment Service for processing.
```

Comment:
If we use the phrase `top up processing` above, we might want to use it here as well.